src/tx-builder/services/DydxGovernance.ts

```diff
@@ -631,4 +631,22 @@ export default class DydxGovernanceService extends BaseService<DydxGovernor> {
       topAgainstVotes,
     };
   }
+
+  public async getGovernanceVoters(
+    governorContractAddress: string,
+    maxBlock: number,
+    startBlock: number = 0,
```

Comment:
small nit: Can you default this to the [deploy block of the `DydxGovernor` contract](https://etherscan.io/tx/0x1cb08c33f96785cad2c86369f214b422c6057ac9bbfc1965bcfccf231de976b4)?

Additionally can you rename `maxBlock` to `endBlock`?